src/compiler/compile/render_dom/wrappers/shared/mark_each_block_bindings.ts

```diff
@@ -10,7 +11,13 @@ export default function mark_each_block_bindings(
 	// we need to ensure that the each block creates a context including
 	// the list and the index, if they're not otherwise referenced
 	binding.expression.references.forEach(name => {
-		const each_block = parent.node.scope.get_owner(name);
+		const each_block = parent.node.scope.get_owner(name) as EachBlock;
+		if (each_block) {
```

Comment:
response of `parent.node.scope.get_owner(name)` can be `EachBlock` / `SlotTemplate` / `InlineComponent` (AFAIK).
So we should check that it is `EachBlock` or not.

```suggestion
		const owner = parent.node.scope.get_owner(name);
		if (owner && owner.type === 'EachBlock') {
```

Reply:
Fixed